handbook/upgrade/upgrade.md

```diff
@@ -0,0 +1,280 @@
+# Upgrading and Migrating SystemLink Server
+
+Upgrading to a new version or migrating between SystemLink Servers is a common practice in the lifecycle of any SystemLink deployment. The goal of this chapter is to complete this operation while retaining a fully operable SystemLink Server. This is critical to mitigate against issues encountered during an upgrade or migration that prevent the new instance of SystemLink Server from being used.
+
+!!!important "Using Virtual Machines for SystemLink Deployments"
+    It is recommended to run SystemLink Server, attached file stores, and databases within virtual machines. Virtual machines simplify the creation and restoration of backups. The use of other tools to facilitate the backup of bare metal machines, e.g.non-virtual machines, is outside the scope of this chapter.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2021 R1 or greater
+
+- Familiarity with installing and setting up a SystemLink Server
+
+    - [Setting up a SystemLink Server](https://www.ni.com/documentation/en/systemlink/latest/setup/)
+
+    - [Configuring NI Web Server](https://www.ni.com/documentation/en/ni-web-server/latest/manual/configuring-ni-web-server/)
+
+        - [Sign on with LDAP](/ldap/ldap)
+
+        - [Single Sign-on with OpenID Connect](/openid-connect/openid-connect/)
+
+    - Data Store Configuration
+
+        - [MongoDB](/data-stores/mongodb/)
+
+        - [Connecting to a Remote Mongo Database](https://www.ni.com/documentation/en/systemlink/latest/setup/remote-mongo-database/)
+
+        - [Connecting to a Remote File Share](https://www.ni.com/documentation/en/systemlink/latest/setup/remote-file-share/)
+
+        - [Uploading Files to Amazon Simple Storage Service (S3)](https://www.ni.com/documentation/en/systemlink/latest/data/uploading-files-to-amazon-s3/)
+
+        - [Connecting to a Remote PostgreSQL Database](https://www.ni.com/documentation/en/systemlink/latest/setup/remote-postgres-databse/)
+
+- Recommended
+
+    - All servers, file stores, and databases running in virtual machines
+
+    - Removable storage volume or device
+
+## Preparing for Upgrade or Migration
+
+Prior to attempting any upgrade or migration you should backup your server and any remote data stores. Failure to create backups increases the risk of unplanned downtime if issues occur during the upgrade or migration process.
```

Comment:
Before you begin, backup your server and any remote data to avoid unplanned downtime if issues occur during the upgrade or migration process.